### DISCUSSION TITLE
New org

### DISCUSSION THREAD

**Discussion by Shluffy** [gosoccerboy5/gosoccerboy5#6] [General]
Would you like to make a new org with me for scratch-consolebased-extensions, previously know as scratch-bookmarklets. I have no control, and just it is hard to actually make something, and you are the only other person who really makes other bookmarklets. If you are interested, I could make an org and invite you and we will do that rather then work as a side of scratch-dev-team.
@gosoccerboy5

**Comment by gosoccerboy5** [comment]
Is there anything wrong with Scratch-ConsoleBased-Extensions? I made a basic PR on SCBE, if you don't like the name, is it possible to rename the organization?

**Reply by Shluffy** [reply]
It's repo. I don't have admin access. I am starting to copy and paste this message in the team chat so he adresses it rather then deletes it.

---------
Also, this is not an 'extension'.

---------

So with all the changes, @ZI-Youtuber you are _NOT_ authorized to change the name without discussion - even if you are admin. This mostly my work, some from @gosoccerboy5 and a little from you, which was a bookmarklet hard to use as who has the project id, it was copy and pasted from the redirecttoscratchuser.js file, and I had to make changes to make it look presentable. 
Even if you are owner of the org, you are not boss of the repo scratch-consolebased-extensions as you barely write code for it. Alos, you said early on that you do not know what to do, so this is  definitely _NOT_ your place to be lording over the repo.

This is no longer something you can ignore.
Please do not:
- Delete This (I assure you it will not get you anywhere)
- Edit this

--Shluffy, Now the monster

CC: @xItzDiamondx @ZI-Youtuber @gosoccerboy5 

-------

Also, my requests for admin permission on this repo have been ignored. Face a problem rather then ignore it. Same with what garbomuffin said on the license for the turbowarp fork. I pinged you many times, but you ignored it.

**Reply by Shluffy** [reply]
and now he invited me back.
